config/locales/components/en.yml

```diff
@@ -39,4 +39,4 @@ en:
     print_button:
       label: Print
     troubleshooting_options:
-      default_heading: 'Having trouble? Here’s what you can do:'
+      default_heading: 'Additional troubleshooting options:'
```

Comment:
Because this is the default heading for the base troubleshooting component, this change will apply to every rendering of troubleshooting options in the entire application. Are we expecting that, or are these changes meant to be limited to in-person proofing?

If we want it to be limited to in-person proofing, we can customize the [`heading`](https://github.com/18F/identity-idp/blob/3c69b3166266d3aba5df3cc208fbb9b98c948761/app/javascript/packages/components/troubleshooting-options.tsx#L17) where we render the troubleshooting options.

For example, InstantVerify verification after these changes:

![image](https://user-images.githubusercontent.com/1779930/205080041-f6db9101-efa9-45c5-b5cd-dae79c44e886.png)